Fix #15506 Update documentation for plugins index.md (#15518)
netbox-community/netbox: docs/plugins/index.md

### Restart WSGI Service

> Restart the WSGI service to load the new plugin and restart rqworkers for using plugin in custom scripts:

Reviewer: Please be sure to use proper grammar and punctuation when suggesting changes to the documentation.
```suggestion
Restart the WSGI service and RQ workers to load the new plugin:
```
